perfkitbenchmarker/configs/benchmark_config_spec.py

```diff
@@ -380,6 +380,12 @@ def _GetOptionDecoderConstructions(cls):
         'high_availability': (option_decoders.BooleanDecoder, {
             'default': False
         }),
+        'backup_enabled': (option_decoders.BooleanDecoder, {
```

Comment:
database_backup_enabled?  but really it feels like maybe we should have a database_spec?